docs/Installation-Script/UnityStationInstaller.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+
+# Fonction de gestion des erreurs
+function handle_error {
+    echo "Une erreur s'est produite. Sortie du script."
+    exit 1
+}
+
+# Fonction pour obtenir la dernière version depuis l'API GitHub
+function get_latest_version {
+    local repo_url="https://api.github.com/repos/unitystation/stationhub/releases/latest"
+    local version=$(wget -qO - "$repo_url" | grep -oP '"tag_name": "\K(.*)(?=")')
+    echo "$version"
+}
+
+latest_version=$(get_latest_version)
+
+while true; do
+    echo "Choose your distribution :"
+    echo "1 - Ubuntu/Debian"
+    echo "2 - Arch-Linux"
+    echo "3 - Fedora"
+    echo "4 - OpenSUSE"
+    echo "5 - CentOS"
+    echo "6 - Gentoo"
+    read -p "Enter the number of your choice : " choice
+
+    case $choice in
+        1)
+            # Commandes pour Ubuntu
+            apt update
+            apt full-upgrade
```

Comment:
as I said earlier, there is no -y 
so the user is free to do **Y/N** if the suggested updates don't suit, it doesn't block the script.


it's more for "the sake of ", to remind the user to do his updates 
don't forget that most users are lazy

Reply:
> it's more for "the sake of ", to remind the user to do his updates

this is an awful justification. This script doesn't need the user to upgrade their system, do not upgrade someone's computer because you think it is best for them. Respect the user.

Reply:
Keeping your system up to date is precisely the first recommendation you should make to a user, especially if they're too lazy to do it.

and i've created this script for people who want everything to work "all-in-one", the flemmes are a parameter to take into account. 

you don't seem convinced?

Reply:
The whole point of a script like this is to do one thing and one thing only, install stationhub.

We don't install firefox automatically because some people are too lazy or force them to install Arch Linux because they're too stupid, because it is not our concern what the user does with their PC and we shouldn't be changing anything about it just because we think its for the best.